templates/repo/diff/comments.tmpl

```diff
@@ -1,6 +1,6 @@
 {{range .comments}}
 
-{{$createdStr:= TimeSinceUnix .CreatedUnix $.root.locale}}
+{{$createdStr:= TimeSinceUnix .CreatedUnix .root.locale}}
```

Comment:
This will cause crash.

When it is in a `range`, it needs `$` to access the top scope.

Reply:
Thanks I missed it